Add EuroBSDCon 2023

mmuman opened the pull request:
Schedule is for now the one I generated myself from the website. Hopefully they'll host it themselves.

Wilm0r commented:
Given the conversation in #294 do you have a XML file for this one now?

Wilm0r commented:
(WTF, justs found this draft comment that actually predates my last one, no idea how that survived in browser/cookie state over days...)

Ha, so I thought I was checking for that, https://github.com/Wilm0r/giggity/blob/9d7869351ec0ff9777be16f34b8038c374d170ef/tools/menu-ci.py#L260 , but I think I failed at my own weird API there since in case of failure it won't return None or whatever, but an exception. (Yes, returns, not throws. Meh.)

.... Anyway, about to fix that now.

Wilm0r commented:
Didn't you say you found an OSEM XML for this now?

mmuman commented:
No they said they used OSEM but the schedule doesn't correspond anyway.

mmuman commented:
I updated and hosted the ics on a Nextcloud server, so we'll get HTTPS.

I added the rooms but it seems they don't work when using ICS, Paris Web also doesn't have clickable room names…

mmuman commented:
ping, it starts today…

mmuman commented:
Thanks!

Wilm0r commented:
Thank you too, as always :D
